CMakeLists.txt

```diff
@@ -5,8 +5,8 @@ SET (boolinq_VERSION_MAJOR 2)
 SET (boolinq_VERSION_MINOR 0)
 
 
-SET (CMAKE_CXX_FLAGS_DEBUG   "${CMAKE_CXX_FLAGS} -O0 -ggdb3 -DDEBUG")
-SET (CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS} -O3")
+SET (CMAKE_CXX_FLAGS_DEBUG   "${CMAKE_CXX_FLAGS} -O0 -ggdb3 -DDEBUG -std=c++11")
+SET (CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS} -O3 -std=c++11")
```

Comment:
@babelvit added this arguments to fix updated google externals/benchmark/test/cxx03_test:
```
set(CXX03_FLAGS "${CMAKE_CXX_FLAGS}")
string(REPLACE "-std=c++11" "-std=c++03" CXX03_FLAGS "${CXX03_FLAGS}")
string(REPLACE "-std=c++0x" "-std=c++03" CXX03_FLAGS "${CXX03_FLAGS}")
```

So I can really use `-std=c++11` or `-std=c++0x`. What do you think?